### DISCUSSION TITLE
Renew screenshots on website

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#2122] [Ideas]
I made some first drafts and my idea was to show some more stuff, not only themes.
Actually on the main page there are shown only 5, this number can be changed.

arch-colors:
![arch-colors](https://user-images.githubusercontent.com/10681413/141349332-158f95f1-bdb0-4a17-8680-c4aea483471d.png)

silver
![silver](https://user-images.githubusercontent.com/10681413/141349325-961b7fdd-1beb-4b77-888a-89b1a49a4b2e.png)

plasma
![plasma](https://user-images.githubusercontent.com/10681413/141349306-3059e6c8-e07a-4087-aa7c-fa9be5e58905.png)

kvantum
![kvantum](https://user-images.githubusercontent.com/10681413/141349315-8d6ebe95-4a26-4b5d-ae78-3cea969277c3.png)

light
![light](https://user-images.githubusercontent.com/10681413/141530686-efac5344-9060-4bfe-85b5-8dd1dab2a0b3.png)

ambiance
![ambiance](https://user-images.githubusercontent.com/10681413/141530307-18172a9c-1024-4f2a-8021-a50999461388.png)

dark
![dark](https://user-images.githubusercontent.com/10681413/141530311-650a6804-5a4d-4293-b91b-d57fd7c1fef1.png)

clearlooks
![clearlooks](https://user-images.githubusercontent.com/10681413/141349303-41e0a400-23de-497e-bf7b-65daab152694.png)

Leech
![leech](https://user-images.githubusercontent.com/10681413/141613458-9a21bc28-b3ca-470c-9b29-0cbd671b0fed.png)

**Comment by tsujan** [comment]
They're very good, especially because they show the users that, even without a widget style like Kvantum, they can have different looks and feels with Qt's default widget style (Fusion)

**Reply by tsujan** [reply]
Please use an icon set that doesn't show white icons on white backgrounds.

EDIT:
Or, perhaps, you could uncheck LXQt Appearance Configuration → Icons Theme → Colorize icons...

EDIT1:
Or, maybe, restarting of apps (e.g., pcmanfm-qt) was needed after changing the palette.

**Reply by stefonarch** [reply]
It's hard to check everything. I'll try. A weird thing is frost theme, it has a dark panel in the VM and  no idea where this results from.

**Reply by tsujan** [reply]
> A weird thing is frost theme, it has a dark panel in the VM and no idea where this results from.

Frost is an interesting theme. It reverses the colors of the current widget style. For example, it uses `palette(text)` for background and `palette(window)` for text. Therefore, if you have a light widget style, Frost will be dark, and conversely.

**Reply by tsujan** [reply]
> It's hard to check everything.

It definitely is. A second pair of eyes is needed, like when you and I fixed the themes.

**Reply by stefonarch** [reply]
I simply forgot to reset the icon set from dark to normal in "dark" and "ambiance".
Added a new one and found naturally some issues in some themes (silver and light).

Your eyes turn now.

**Reply by tsujan** [reply]
All look good. They show different settings and different possibilities.

**Reply by stefonarch** [reply]
Added a last one for Leech and qps.

Sorry if I use this here for reminders but 2 things I noticed:
* panel settings' height is at the limit here with 768 px, on smaller/older screen they don't fit and I saw an user on twitter who didn't manage to click apply (using alt +drag or a smaller fontsize)
* dropping apps from menu to quicklaunch or desktop could show an + sign when in the right place.

**Reply by tsujan** [reply]
> panel settings' height is at the limit here with 768 px

I noticed it in your screenshots. Yes, it should be made scrollable, like QTerminal's settings dialog.

> dropping apps from menu to quicklaunch or desktop could show an + sign when in the right place.

That made me think that Quick Launch might have  a problem at first. But I forgot about it because I arranged the contents once for all, on an auto-hiding panel.

**Reply by stefonarch** [reply]
Last question is about the titles, remove "LXQt" from them? It's somehow redundant. Or Ambiance theme, Silver theme ecc?
![schermata-11-13-14-18](https://user-images.githubusercontent.com/10681413/141645477-317b63f9-8fd5-4a5c-873f-c3404d9a8690.png)

**Reply by tsujan** [reply]
> remove "LXQt" from them?

I think it isn't redundant. It shows that the themes are *LXQt* themes. For example, "LXQt Kvantum" isn't mistaken for the Kvantum widget style.

**Reply by stefonarch** [reply]
Ok, and the very last thing: some text (and if yes what) or not?
![schermata-11-13-14-53](https://user-images.githubusercontent.com/10681413/141646440-a428c0e3-4cd9-4ed8-95fa-6a634d5fbcdc.png)

**Reply by tsujan** [reply]
To me, it's very good.

**Reply by tsujan** [reply]
This isn't important but why disabling compositing with themes that support translucency? LXQt Kvantum supports it but its screenshot doesn't show that.

**Reply by stefonarch** [reply]
Don't ask my why but compton refused to start at some point... will remake this one.
And add a link to  https://github.com/lxqt/lxqt/wiki/Theming in  the text.

**Reply by tsujan** [reply]
I especially like what you've done with the palette in your first screenshot on this page.

It doesn't seem to be Fusion; is it?

**Reply by stefonarch** [reply]
Thanks! Frankly I don't remember, at some point I used only fusion, this is with fusion:
![schermata-11-13-19-11](https://user-images.githubusercontent.com/10681413/141654625-0d8de5e6-292e-4258-a371-0083a06e9705.png)

And kvantum with compositor running:
![kvantum](https://user-images.githubusercontent.com/10681413/141654634-722afefe-b047-4001-bfe0-d8e221bcd36f.png)

**Reply by tsujan** [reply]
> I used only fusion, this is with fusion

Strange! I didn't know that Fusion can have a light toolbar with a dark window.

> And kvantum with compositor running:

I'm sure LXQt Kvantum's menus aren't so translucent. Something should have increased its translucency in your screenshot. The line-edit (inside Main Menu) is totally opaque in the theme.

BTW, if you find the time, you could use https://github.com/lxqt/lxqt-panel/pull/1681 and get rid of that huge config dialog. It'll be a test too.

**Reply by stefonarch** [reply]
>BTW, if you find the time, you could use lxqt/lxqt-panel#1681 and get rid of that huge config dialog. It'll be a test too.

I thought a moment to do so, but we shouldn't show newer things - or make a point release for the panel?
And yes, you're right about the transparency - it has to be the half transparent panel, it affect also the menu.

**Reply by stefonarch** [reply]
edit, no, it was compton settings 80% transparency for menus

**Reply by stefonarch** [reply]
![kvantum](https://user-images.githubusercontent.com/10681413/141655510-11414f80-9d3a-48ea-b1bf-6d8918f0c5c4.png)

**Reply by tsujan** [reply]
> I thought a moment to do so, but we shouldn't show newer things - or make a point release for the panel?

Point releases are only for bad bugs. With the discussion you started at https://github.com/lxqt/lxqt-panel/issues/1680, I think it's OK not to make any change here.

> it was compton settings 80% transparency for menus

Now the menu is fine :)
